ietf/iesg/tests.py

```diff
@@ -96,6 +96,12 @@ def test_photos(self):
         ads = Role.objects.filter(group__type='area', group__state='active', name_id='ad')
         self.assertEqual(len(q('.photo')), ads.count())
         
+    def test_ietf_activity(self):
```

Comment:
I'll take this as is, but please start making the tests more meaningful than "the page doesn't crash".